src/lbpDashboard/sparkChart/spark-chart.ts

```diff
@@ -21,17 +21,14 @@ export class SparkChart {
   }
 
   attached(): void {
-    if (this.chart) {
-      this.chart.resize(this.container.offsetWidth, this.container.offsetHeight);
-    }
+    if (this.chart) this.resizeChart();
+
     window.onresize = () => {
       /**
        * don't resize when the element is hidden, or height will go permanently to 0
        */
       setTimeout(() => {
-        if (this.chart) {
-          this.chart.resize(this.container.offsetWidth, this.container.offsetHeight);
-        }
+        if (this.chart) this.resizeChart();
```

Comment:
Best practice is to put everything in braces.  (Ought to be a lint rule for that).  Just for future reference....